Descriptors/EvaluationTypeDescriptor.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<InterchangeDescriptors xmlns="http://ed-fi.org/3.3.1-b" xmlns:ann="http://ed-fi.org/annotation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ed-fi.org/3.3.1-b ../Schemas/Bulk/EXTENSION-Interchange-Descriptors-Extension.xsd">
+	<EvaluationTypeDescriptor>
+		<CodeValue>Formal Eval</CodeValue>
+		<ShortDescription>Formal Eval</ShortDescription>
+		<Description>Formal evaluation</Description>
+		<Namespace>uri://tpdm.ed-fi.org/EvaluationTypeDescriptor</Namespace>
+	</EvaluationTypeDescriptor>
+	<EvaluationTypeDescriptor>
+		<CodeValue>Formal Obs</CodeValue>
+		<ShortDescription>Formal Obs</ShortDescription>
+		<Description>Formal Observation</Description>
+		<Namespace>uri://tpdm.ed-fi.org/EvaluationTypeDescriptor</Namespace>
+	</EvaluationTypeDescriptor>
+	<EvaluationTypeDescriptor>
+		<CodeValue>Self Eval</CodeValue>
+		<ShortDescription>Self Eval</ShortDescription>
+		<Description>Formal evaluation self-rating</Description>
+		<Namespace>uri://tpdm.ed-fi.org/EvaluationTypeDescriptor</Namespace>
+	</EvaluationTypeDescriptor>
+	<EvaluationTypeDescriptor>
+		<CodeValue>Informal Obs</CodeValue>
+		<ShortDescription>Informal Obs</ShortDescription>
+		<Description>Informal Observation</Description>
+		<Namespace>uri://tpdm.ed-fi.org/EvaluationTypeDescriptor</Namespace>
+	</EvaluationTypeDescriptor>
+	<EvaluationTypeDescriptor>
+		<CodeValue>Student Growth</CodeValue>
+		<ShortDescription>Student Growth</ShortDescription>
+		<Description>Student Growth Measures</Description>
+		<Namespace>uri://tpdm.ed-fi.org/EvaluationTypeDescriptor</Namespace>
+	</EvaluationTypeDescriptor>
+	<EvaluationTypeDescriptor>
+		<CodeValue>Self Informal Obs</CodeValue>
```

Comment:
Since Informal Observation self-rating is well under the 75 char limit, would it make sense to just have this be the CodeValue and ShortDescription as well?